packages/slice-machine/server/src/api/common/LibrariesState.ts

```diff
@@ -0,0 +1,72 @@
+import Files from "@lib/utils/files";
+import Environment from "@lib/models/common/Environment";
+import * as Core from '@slicemachine/core'
+
+export function generateState(env: Environment): void {
+  const libraries = (env.userConfig.libraries || [])
+    .map(lib => Core.Libraries.handleLibraryPath(env.cwd, lib))
+    .filter(Boolean) as ReadonlyArray<Core.Models.Library.Library>;
+
+  const state = formatLibraries(libraries);
```

Comment:
A bit weird, I would expect the type of State to be LibraryState and not LibraryState.library